.gitignore

```diff
@@ -2,13 +2,15 @@
 *~
 *.backup
 /.dir-locals.el
+.DS_Store
 *.orig
 /.project
 .settings
 *.sw[a-p]
 .vscode
 \#*\#
 .\#*
+.idea/
```

Comment:
Just curious about which tool generates this folder.

Reply:
My IDEs: CLion and PyCharm. :-)